prime-router/src/main/kotlin/cli/TestReportStream.kt

```diff
@@ -507,6 +507,8 @@ class End2End : CoolTest() {
             val tree = jacksonObjectMapper().readTree(json)
             val reportId = ReportId.fromString(tree["id"].textValue())
             echo("Id of submitted report: $reportId")
+            val topic = tree["topic"].textValue()
+            echo("Topic for client org sender: $topic")
```

Comment:
Can you change this so that it fails if the topic is not exactly 'covid-19'? - this is a stronger test than just the exception getting thrown.   Look for the pattern in other places in this file with a 'passed' boolean:
   `var passed = true`
Then you set it to false on error, but allow the test to continue, and return that value from the test.   Make sure once its false it never gets reset to 'true'.   

In addition, there is a method `bad` that prints stuff in red, for failure, and a method `good` that prints stuff in green, for a test passing.   So, when a failure does occur, you can easily see what's going on.

Thanks!